controller/resource/csp_billing_adapter.go

```diff
@@ -75,9 +75,9 @@ func GetCspConfig() api.RESTFedCspSupportResp {
 
 	if obj, err = global.ORCH.GetResource(RscTypeConfigMap, NvAdmSvcNamespace, "metering-archive"); err == nil {
 		if cm, ok := obj.(*corev1.ConfigMap); cm == nil || !ok {
-			err = fmt.Errorf("Error: Unknown type")
+			log.WithFields(log.Fields{"err": "Error: Unknown type"}).Error()
```

Comment:
same issue like line 45

Reply:
I restore this change already, this is an valid assignment.